src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -47,26 +49,50 @@ class FeelingRoom extends Component {
     floatAnimation('#fineCircle12', 1)
     return (
       <section className='fineRoom'>
-        <div className='fineCircles' id='fineCircle1'></div>
-        <div className='fineCircles' id='fineCircle2'></div>
-        <div className='fineCircles' id='fineCircle3'></div>
-        <div className='fineCircles' id='fineCircle4'></div>
-        <div className='fineCircles' id='fineCircle5'></div>
-        <div className='fineCircles' id='fineCircle6'></div>
-        <div className='fineCircles' id='fineCircle7'></div>
-        <div className='fineCircles' id='fineCircle8'></div>
-        <div className='fineCircles' id='fineCircle9'></div>
-        <div className='fineCircles' id='fineCircle10'></div>
-        <div className='fineCircles' id='fineCircle11'></div>
-        <div className='fineCircles' id='fineCircle12'></div>
+        <section className='hexarea'>
+          <div className='fineCircles' id='fineCircle1'></div>
+          <div className='fineCircles' id='fineCircle2'></div>
+          <div className='fineCircles' id='fineCircle3'></div>
+          <div className='fineCircles' id='fineCircle4'></div>
+          <div className='fineCircles' id='fineCircle5'></div>
+          <div className='fineCircles' id='fineCircle6'></div>
+          <div className='fineCircles' id='fineCircle7'></div>
+          <div className='fineCircles' id='fineCircle8'></div>
+          <div className='fineCircles' id='fineCircle9'></div>
+          <div className='fineCircles' id='fineCircle10'></div>
+          <div className='fineCircles' id='fineCircle11'></div>
+          <div className='fineCircles' id='fineCircle12'></div>
+        </section>
       </section>
     )
   }
   createAnxiousRoom = () => {
-    //functions to animate go here
+    anxiousAnimation('#hex1', .74, 1, 1.8, .90)
+    anxiousAnimation('#hex2', .5, 1, .7, .45)
+    anxiousAnimation('#hex3', .6, .7, 3.4, 2)
+    anxiousAnimation('#hex4', .78, .2, .2, .75)
+    anxiousAnimation('#hex5', .67, .1, .1, 0)
+    anxiousAnimation('#hex6', .74, 1, 1.8, .90)
+    anxiousAnimation('#hex7', .86, 1, 1.8, .55)
+    anxiousAnimation('#hex8', .8, .5, 2, 1.5)
+    anxiousAnimation('#hex9', .5, 1, .7, .45)
+    anxiousAnimation('#hex10', .7, 1, 2.5, 2.5)
+    anxiousAnimation('#hex11', .49, .5, .3, 2.5)
+
     return (
-      console.log('ANXIOUS')
-      //elements to animate go here
+      <section className='anxiousRoom'>
```

Comment:
Looking good, but a little long. Might have a refactor for all of these down the line.

Reply:
Agreed! I'd love to see if we could refactor this to be a bit shorter